app/services/UIBuilder.scala

```diff
@@ -189,4 +191,11 @@ class UIBuilder {
     val texts: Seq[Text] = ycg.group.map(c => TextBuilder.fromPhrase(c.text))
     ConfirmationPanel(texts.head, texts.tail)
   }
+  private def fromSectionAndNoteGroup(caption: Text, ng: NoteGroup)(implicit stanzaIdToUrlMap: Map[String, String]): UIComponent = {
```

Comment:
Dont need the extra braces on these two fns

Reply:
removed